client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -104,9 +107,21 @@ def get_pre_create_attr_defs(self):
                                         "FBX unit of centimeters.",
                                 default=False,
                                 label="Convert Units")
-
+        format_ASCII = BoolDef("exportkind",
```

Comment:
Variables should be lower cased.
```suggestion
        format_ascii = BoolDef("exportkind",
```